liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -70,15 +72,14 @@
 import io.openliberty.tools.common.plugins.config.ServerConfigXmlDocument;
 import io.openliberty.tools.maven.ServerFeatureSupport;
 import io.openliberty.tools.maven.applications.LooseWarApplication;
+import io.openliberty.tools.maven.utils.DevHelper;
```

Comment:
Given you removed several things from this class, I would expect there to be fewer `import` statements not more. Are these new ones needed?

Reply:
Nope - removed.